pybrake/test_notifier.py

```diff
@@ -135,7 +138,8 @@ def test_unknown_host():
 
   notice = notifier.notify_sync('hello')
 
-  assert str(notice['error']) == '<urlopen error [Errno -2] Name or service not known>'
+  assert isinstance(notice['error'], URLError)
+  assert 'not known>' in str(notice['error'])
```

Comment:
Since this wraps an `OSError` the OS message bubbles up and may be different across platforms. For me on High Sierra:

```
urllib.error.URLError: <urlopen error [Errno 8] nodename nor servname provided, or not known>
```